### PR TITLE
Fix nls_test(): Only check the first 20 bytes of output

### PR DESCRIPTION
**Pull request by LeoHsiao1**

### CODE REVIEW

**Comment by clanmills** [comment]
This is outstanding work, @LeoHsiao1.  I love the `[:20].decode()`.  So elegant.  And the `platform=` and `nls_enable=` code is beautifully simple and clear.

Three points (because there are always discussion points).

1. CI (Travis and Appveyor)
- This isn't tested by travis because -DEXIV2_ENABLE_NLS=1 has not been set.
- I suspect the same is true with Appveyor
- Can you update `.travis.yml` and `appveyor.yml`

2. When -DEXIV2_ENABLE_NLS=1
- You fail because the reference output has not been changed.

3. Updating test/nls-test.sh.
- Add the `| head -c 20` code to test/nls-test.sh 
- Update the reference file test/data/nls-test.out

_**How to proceed**_

1. Can you do PRs in future in https://github.com/exiv2/exiv2 because I can update the PR.

2. Can you deal with the three points above and update this PR.
I know you're not overly confident in bash, however I would like you to be "King Test" and own the test suite.  The bash scripts in test/*.sh are heading for the firing squad before the end of 2020.  Last chance for you to learn more about bash.  And _**important for me**_, it gives me more time for the book.

3. About #1360
You don't need to do anything about testing Exiv2::base64decode (or Exiv2::base64encode).  Luis has offered to add to unit_tests for that.  I was astonished when I updated #1360 and the mergify robot merged and closed the PR.  As you know, I'm not strong in git/github.  I'll discuss with Luis about removing the bot.  With friends like mergify, who needs enemies?

_**Bottom Line**_

Thank for working on this, Mr Lion.  I'm very pleased indeed with your contributions.   Even better, I appreciate your _**just do it**_ attitude.

**Comment by clanmills** [comment]
@LeoHsiao1   Can you apply patch below to your PR.

That fixes test/nls-test.sh.  However there's something not quite right in testcases.py.  The bash syntax `|head -c 20` pipes the first 20 bytes to stdout and nothing else.  There's something not right testcases.py or nls-test.sh. 

```
diff --git a/test/data/nls-test.out b/test/data/nls-test.out
index 9312e3f..bf3a148 100644
--- a/test/data/nls-test.out
+++ b/test/data/nls-test.out
@@ -1,10 +1,5 @@
 exiv2: Une action doit être spécifié
 exiv2: Au moins un fichier est nécessaire
-Utilisation : exiv2  [ options ] [ action ] fichier ...
-
-Manipulation des métadonnées EXIF issues des images.
-exiv2: Se debe especificar una acción
+Utilisation : exiv2 exiv2: Se debe especificar una acción
 exiv2: Se requiere un archivo al menos
-Uso: exiv2 [ opciones ] [ acción ] archivo ...
-
-manipular los metadatos Exif de las imágenes.
+Uso: exiv2 [ opcione
\ No newline at end of file
diff --git a/test/nls-test.sh b/test/nls-test.sh
index 7948d30..7048603 100755
--- a/test/nls-test.sh
+++ b/test/nls-test.sh
@@ -38,7 +38,7 @@ source ./functions.source
 	for l in fr_FR es_ES; do (
 		export LC_ALL=$l
 		export $LANG=$l 
-		runTest exiv2
+		runTest exiv2 | head -c 20
     ) done
 
 ) 3>&1 > $results 2>&1
```

I'm sure you'll find it easier to work on a branch in exiv2/exiv2.git

**Comment by LeoHsiao1** [comment]
I have resubmitted the PR, please discuss it in #1371 .
